vpr/src/base/echo_files.cpp

```diff
@@ -158,9 +160,9 @@ void alloc_and_load_output_file_names(const std::string default_name) {
     char* name;
 
     if (outputFileNames == nullptr) {
-        outputFileNames = (char**)vtr::calloc((int)E_FILE_END_TOKEN, sizeof(char*));
+        outputFileNames = new char*[(int)E_FILE_END_TOKEN]();
```

Comment:
Again, best to replace calloc with explicit 0 initialization (or find documentation in the standard saying this is guaranteed to set to 0, and comment that and the fact that we rely on it).